### DISCUSSION TITLE
Vercel WooCommerce Integration.

### DISCUSSION THREAD

**Discussion by chrislemmer** [vercel/community#967] [Show & Tell]
Hi everyone,

I was looking through the available Vercel integrations and noticed that nobody has created an integration for WooCommerce.

I was curious as to why. Does anyone know of specific reasons?

Thanks on advance.
Chris

**Comment by amyegan** [comment]
Great question @chrislemmer! WooCommerce is very tightly coupled with WordPress. That doesn't mean you couldn't find a way to set it up to work for your app, but it would be more difficult to turn into a Vercel integration. 

If you're interested in other options, there are some good alternative [commerce integrations](https://vercel.com/integrations#commerce) available in the [Integrations Marketplace](https://vercel.com/integrations).

**Reply by chrislemmer** [reply]
> Great question @chrislemmer! WooCommerce is very tightly coupled with WordPress. That doesn't mean you couldn't find a way to set it up to work for your app, but it would be more difficult to turn into a Vercel integration.
> 
> If you're interested in other options, there are some good alternative [commerce integrations](https://vercel.com/integrations#commerce) available in the [Integrations Marketplace](https://vercel.com/integrations).

Thanks for the response!

If an integration was built, do you think people would use it? I wonder whether developers would even want to build headless sites with WooCommerce. Or would they rather use another stack in the backend?

Perhaps it's useful to existing WooCommerce sites that have reached a ceiling with the monolithic architecture but don't want to migrate to a new stack due to investment.

I would love to hear your thoughts.
